### DISCUSSION TITLE
Is it possible to download the top hit for either all open tabs or the current tab with a keyboard shortcut?

### DISCUSSION THREAD

**Discussion by Kamacks** [aclap-dev/video-downloadhelper#424] [Help and Q&A]
### Your question

Using VDH on windows 10 with the companion app. I have it installed on Chrome and Firefox.

I have searched google/reddit/this page but couldn't find anything that really would work for me. Apologies if this is trivial/already been answered. 

To be a little more in depth I'm trying to bypass the monotonous repetition of the same basic clicks over and over when I have multiple files I'd like to download. 

I'd like to push a hotkey (or click a button) that automatically starts downloading the top shown .mp4 file result from clicking the extension icon or right clicking the page (or possibly matching specific properties of a file?).

If there is a way to do this for ALL open tabs that have media on them on a specific site that would be preferable, but even just having the ability to have a keyboard shortcut for the current tab I'm on to automatically select the file and start it downloading would be amazing if not. 

Hopefully this makes sense! Thanks!!

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
Hi, sorry it's not possible to download from all open tabs in one shot.

**Comment by GrampaWildWilly** [comment]
There might be something you could use, but you'd have to experiment with it to see whether it's at least in the direction of what you want.  In Firefox (I don't have Chrome), do Ctrl+Shift+a.  This puts you on the Manage Your Extensions page.  At the top of the page there is a gear icon.  Click that.

![image](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/59ca46ed-b394-4dc9-8a2b-bd861c3ff4c5)

Click the Manage Extension Shortcuts entry on that menu.  Experiment with what that gives you.
